types/glob/index.d.ts

```diff
@@ -20,6 +20,7 @@ declare namespace G {
 
     function hasMagic(pattern: string, options?: IOptions): boolean;
 
+    let glob: typeof G;
```

Comment:
Should this be `typeof G` or `IGlobStatic`?

Reply:
@rbuckton thx for the review!
I believe it should be the module itself (`typeof G`), this serves as backward compatible export:
https://github.com/isaacs/node-glob/blob/v1.0.0/lib/glob.js#L5
Thanks!